straxen/plugins/veto_hitlets.py

```diff
@@ -120,6 +122,23 @@ def compute(self, records_nv, start, end):
         return hitlets
 
 
+def remove_switched_off_channels(hits, channel_range, to_pe):
+    """Removes hits which were found in a channel without any gain.
+
+    :param hits: Hits found in records.
+    :param channel_range: Tuple of min/max channel. E.g. (2000, 2119)
+        for the neutron-veto.
+    :param to_pe: conversion factor from ADC per sample.
+    :return: Hits
+    """
+    channel_on = np.argwhere(to_pe > 0).flatten()
+    all_channel = np.arange(channel_range[0], channel_range[1] + 1)
```

Comment:
By the way, you have an off by one here, since https://github.com/XENONnT/straxen/blob/0fa353810faab6b7b80a9bc0dbf7d4c8835bb993/straxen/plugins/veto_hitlets.py#L78 is end inclusive right?